GIS/location.py

```diff
@@ -0,0 +1,22 @@
+import json
```

Comment:
Can you please move these scripts to the `scipts/` folder to keep these at once place?

Reply:
Sure

Reply:
Where is this folder